traffic_portal/app/src/common/modules/form/deliveryServiceStaticDnsEntry/form.deliveryServiceStaticDnsEntry.tpl.html

```diff
@@ -31,42 +31,42 @@
         <br>
         <form name="dsStaticDnsEntryForm" class="form-horizontal form-label-left" novalidate>
             <div class="form-group" ng-class="{'has-error': hasError(dsStaticDnsEntryForm.host), 'has-feedback': hasError(dsStaticDnsEntryForm.host)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">Host *</label>
```

Comment:
why removing the * on required fields?

Reply:
Because they're already required by virtue of the `required` attribute, as well as the big red "Required" we put under them when they aren't filled in. If the `<form>`s submitting `<button>` were not disabled until these requirements were met, the user-agent would also refuse to submit the form with an appropriate message (e.g. "Please fill out this field" in Firefox - I think Chrome says "This field is required").

Reply:
yeah the `*` seems to be common throughout TP for required fields

Reply:
I think I could be on board with removing the `*` for required fields. However, these changes are conflating the PR which should generally just be a single related change.

I think at one point the required fields weren't outlined in red when you opened the page, so having the `*` was helpful from a UI perspective. Does that sound right @mitchell852? Since it now has a better visual indication of required fields I'm inclined to agree with them being unnecessary now.

Reply:
No, I agree, any changes to those should be uniformly applied across TP forms - and thus doesn't belong in this PR. I've just been waiting for a conversation on a separate PR where I'd done the same thing to come to a conclusion before I did anything on this one.

FYI an extended discussion about the future of ` *` could be had on Issue #3381 that I opened to address my thoughts on the matter.